### DISCUSSION TITLE
V23_RC2虚拟机切换分辨率，系统黑屏

### DISCUSSION THREAD

**Discussion by dheart-joe** [linuxdeepin/developer-center#10377] [Developer Discussion | 开发者讨论]
1、在qemu上安装V23 RC2版本的虚拟机，切换分辨率偶尔会导致系统黑屏。
![image](https://github.com/user-attachments/assets/bb9b3081-3c1c-4334-a647-ad36a40be79b)

2、能在宿主机ssh登陆到虚拟机，系统负载正常
![image](https://github.com/user-attachments/assets/244e6d5e-a8e6-435a-888c-d2ee57bea11c)

3、查看xsession-errors的日志有报错
![image](https://github.com/user-attachments/assets/d4e335ba-8e12-4fdf-b58a-6719b05f5d6b)

**Comment by BLumia** [comment]
更到最新内测了吗？rc2 和最新内测还是有一些差距的，包含很多缺陷修复。或者可以等两天试正式版镜像看看是否有相同问题。

**Reply by dheart-joe** [reply]
没有更新到内侧，没有申请内侧资格，过两天再观察看看。
